src/test/kotlin/org/jitsi/utils/stats/BucketStatsTest.kt

```diff
@@ -50,21 +62,47 @@ class BucketStatsTest : ShouldSpec() {
                 bucketStats.toJson()["max_delay_ms"] shouldBe 100
             }
 
-            should("export the buckets correctly to json") {
+            context("export the buckets correctly to json") {
+                bucketStats.addValue(-100)
                 repeat(100) { bucketStats.addValue(1) }
                 repeat(100) { bucketStats.addValue(5) }
                 bucketStats.addValue(150)
                 bucketStats.addValue(1500)
-                val bucketsJson = bucketStats.toJson()["buckets"]
-                bucketsJson.shouldBeInstanceOf<OrderedJsonObject>()
-
-                bucketsJson["0_ms"] shouldBe 0
-                bucketsJson["1_to_2_ms"] shouldBe 100
-                bucketsJson["3_ms"] shouldBe 0
-                bucketsJson["4_to_5_ms"] shouldBe 100
-                bucketsJson["6_to_200_ms"] shouldBe 1
-                bucketsJson["201_to_999_ms"] shouldBe 0
-                bucketsJson["1000_to_max_ms"] shouldBe 1
+
+                context("In separate buckets (default)") {
+                    val bucketsJson = bucketStats.toJson()["buckets"]
+                    bucketsJson.shouldBeInstanceOf<OrderedJsonObject>()
+
+                    bucketsJson["min_to_0_ms"] shouldBe 1
```

Comment:
For items that are inherently unsigned (like number of participants in a conference, having "min_to_0" as the first bucket is a bit weird?

Reply:
I changed it to not automatically add Long.MIN_VALUE and Long.MAX_VALUE to the intervals, so we can start at 0 in such cases.